app/serializers/discussion_serializer.rb

```diff
@@ -1,10 +1,14 @@
 class DiscussionSerializer < EditableSerializer
   attributes :id, :title, :subtitle, :body, :open,
-    :url, :comments_count, :created_at, :updated_at, :status,
+    :url, :comments_count, :commenter_ids, :created_at, :updated_at, :status,
     :tag_list
 
   has_one :author, embed: :ids
 
+  def commenter_ids
+    object.comments.pluck(:author_id).uniq
```

Comment:
Does active_record allow to do
`object.comments.author_ids.uniq` ?

Another thing it might be better to let the DB do the distinct part (if this is even possible, didn't try)
`object.comments.pluck('DISTINCT author_id')`